catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/AssociationsAttributes.java

```diff
@@ -26,23 +27,25 @@
  * This class provides attributes that represent associations between products.
  */
 public class AssociationsAttributes implements Associations, MetacardType {
-    private static final Set<AttributeDescriptor> DESCRIPTORS = new HashSet<>();
+    private static final Set<AttributeDescriptor> DESCRIPTORS;
 
     private static final String NAME = "associations";
 
     static {
-        DESCRIPTORS.add(new AttributeDescriptorImpl(DERIVED,
+        HashSet<AttributeDescriptor> descriptors = new HashSet<>();
```

Comment:
No reason other than a personal preference for concrete types on locally-scoped references. You're right though, using the interface is best practice, I don't mind fixing it.